packages/mui-system/src/Unstable_Grid/GridProps.ts

```diff
@@ -161,7 +161,18 @@ export interface GridBaseProps extends Breakpoints {
 }
 
 export interface GridOwnerState extends GridBaseProps {
-  nested: boolean;
```

Comment:
Changed to `level` sounds better because it is not a boolean anymore, it is a number.